ruby.rb

```diff
@@ -0,0 +1,19 @@
+class Hound
+
+  def hello
+    'hello'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.